src/components/runbooks/RunbookForm.js

```diff
@@ -0,0 +1,202 @@
+import React, {Component} from 'react';
+import {observer} from 'mobx-react';
+import cx from 'classnames';
+import FileSaver from 'file-saver';
+
+import FileInput from '../FileInput';
+
+@observer
+export default class RunbookForm extends Component {
+  changeParameter = (index, attr, value) => {
+    let parameters = this.props.data.parameters.slice();
+    parameters[index][attr] = value;
```

Comment:
Why do you clone parameters at all, if you're mutating the value directly anyway and it is mutated in the old array also?